src/linear_solver_logs/solve_log_ma.jl

```diff
@@ -25,21 +36,28 @@ mutable struct MAInfo
 end
 
 """ 
-    DistInfo
+    SEDistInfo
 
 A mutable structure that stores information about the sub-Exponential distribution.
 
 # Fields
-- `sampler::Union{DataType, Nothing}`, The type of sampler being used.
-- `max_dimension::Int64`, The dimension that is being sampled.
-- `block_dimension::Int64`, The sampling dimension.
-- `sigma2::Union{Float64, Nothing}`, The variance parameter in the sub-Exponential distribution, 
+- `sampler::Union{DataType, Nothing}`, the type of sampler being used.
+- `max_dimension::Int64`, the dimension that is being sampled.
+- `block_dimension::Int64`, the sampling dimension.
+- `sigma2::Union{Float64, Nothing}`, the variance parameter in the sub-Exponential distribution, 
    if not given is determined for sampling method.
-- `omega::Union{Float64, Nothing}`, The exponential distrbution parameter, if not given is determined for sampling methods.
-- `eta::Float64`, A parameter for adjusting the conservativeness of the distribution, higher value means a less conservative
-  estimate. By default, this is set to one.
+- `omega::Union{Float64, Nothing}`, the exponential distrbution parameter, if not given is determined for sampling methods.
+- `eta::Float64`, a parameter for adjusting the conservativeness of the distribution, higher value means a less conservative
+  estimate. By default, this is set to `1`.
+
+For more information see:
+Pritchard, Nathaniel, and Patel, Vivak. Solving, tracking and stopping streaming linear inverse problems. 
+Inverse Problems 40.8 Web. doi:10.1088/1361-6420/ad5583.
+
+Pritchard, Nathaniel and Vivak Patel. “Towards Practical Large-Scale Randomized Iterative Least Squares 
+Solvers through Uncertainty Quantification.” SIAM/ASA J. Uncertain. Quantification 11 (2022): 996-1024.
```

Comment:
Different formats. Should be a markdown list.

Reply:
I have made the format the same